src/isomorphic/children/__tests__/onlyChild-test.js

```diff
@@ -94,4 +94,12 @@ describe('onlyChild', () => {
     }).not.toThrow();
   });
 
+  it('should return the only child', () => {
+    var instance =
```

Comment:
You can fix this by running

```
git reset --hard origin/master
```

then cherry picking these two commits you want, and then force pushing.